src/Classes/Entities/ApplicantEntity.php

```diff
@@ -268,4 +290,15 @@ public function getCohortDate()
     {
         return date("F, Y", strtotime($this->cohortDate));
     }
+
+    /**
+     * Gets cohort date as a yyyymmdd
+     * @return false|string
+     */
+    public function getCohortDateNumber()
```

Comment:
poorly named function, can we follow the same convention as the other date methods and put the format on the method name